package.json

```diff
@@ -8,6 +8,10 @@
     "dev": "nodemon src/index.ts",
     "start": "ts-node src/index.ts"
   },
+  "eslintConfig": {
+    "extends": "@snapshot-labs"
+  },
+  "prettier": "@snapshot-labs/prettier-config",
```

Comment:
@bonustrack When i run 
```
yarn prettier --write .
```
I get this error:
![image](https://user-images.githubusercontent.com/15967809/205299815-df85beec-d404-45e5-be9c-140ef99c84f2.png)

You see them too? should we add this command to package.json similar to `lint` ?

Reply:
@ChaituVR prettier is setup with eslint, we shouldn't use prettier directly